database/redis/database_test.go

```diff
@@ -5,24 +5,13 @@ import (
 
 	"github.com/moira-alert/moira/logging/go-logging"
 	. "github.com/smartystreets/goconvey/convey"
-
-	"github.com/moira-alert/moira"
 )
 
-var config = Config{Port: "6379", Host: "0.0.0.0"}
-var emptyConfig = Config{}
-var testSource = DBSource("test")
-
-// use it only for tests
-func newTestDatabase(logger moira.Logger, config Config) *DbConnector {
-	return NewDatabase(logger, config, testSource)
-}
-
```

Comment:
I am not sure that we should remove this configs from test files and place in database.go. I think that we need just to place this configs in both modules that use it. Of course this is copy-paste and do not follow that DRY principle but I think that it's ok for tests. Placing this configs in database.go can lead to misunderstand the purpose of this configs by contributors and using it in another places of codebase.

Reply:
fixed